### PR TITLE
Video grabber example fixes

### PR DESCRIPTION
**Pull request by tpltnt**
Hi,

i changed the coding style of the video grabber example according to the [oF coding style guide](https://github.com/openframeworks/openFrameworks/wiki/oF-code-style).
Cheers
tpltnt

### CODE REVIEW

File: examples/video/videoGrabberExample/src/ofApp.cpp [hunk 1]
```diff
@@ -2,123 +2,106 @@
...
+    camWidth = 320;  // try to grab at this size.
+    camHeight = 240;
+
+    //we can now get back a list of devices.
+    vector<ofVideoDevice> devices = vidGrabber.listDevices();
+
     for(int i = 0; i < devices.size(); i++){
-		cout << devices[i].id << ": " << devices[i].deviceName; 
-        if( devices[i].bAvailable ){
+        cout << devices[i].id << ": " << devices[i].deviceName;
```
Comment (bakercp) [inline examples/video/videoGrabberExample/src/ofApp.cpp]: I realize this was already here, but seems odd that we still have a `std::cout` in the code.  Wouldn't it be better to wrap it up in an `ofLog*`?

Comment (bakercp) [inline examples/video/videoGrabberExample/src/ofApp.cpp]: Wait, just realized this was the example, so it's probably ok ... (?).

**Comment by tpltnt** [comment]
@bakercp good call, i adjusted it.

**Comment by bakercp** [comment]
Cool.  Seems like the right way to go.

**Comment by kylemcdonald** [comment]
:+1:

File: examples/video/videoGrabberExample/src/ofApp.cpp [hunk 2]
```diff
@@ -2,123 +2,105 @@
...
+    camHeight = 240;
+
+    //we can now get back a list of devices.
+    vector<ofVideoDevice> devices = vidGrabber.listDevices();
+
     for(int i = 0; i < devices.size(); i++){
-		cout << devices[i].id << ": " << devices[i].deviceName; 
-        if( devices[i].bAvailable ){
-            cout << endl;
+        if(cams[i].bAvailable){
```
Comment (ofTheo) [inline examples/video/videoGrabberExample/src/ofApp.cpp]: where does cams[i] come from? shouldn't it be devices? 
did you try and compile this before submitting this PR?

Comment (tpltnt) [inline examples/video/videoGrabberExample/src/ofApp.cpp]: @ofTheo no, i did not. I was editing another file with similar code and it resulted in an copypasta error. Good catch, thank you.

Comment (bilderbuchi) [inline examples/video/videoGrabberExample/src/ofApp.cpp]: compiling your changes should really be the absolute minimum of testing one does on a PR. ;-)

**Comment by tpltnt** [comment]
@bilderbuchi @ofTheo it does compile now, i just checked. Sorry for screwing up on my end.

**Comment by arturoc** [comment]
thanks!

**Pull request merged**